### DISCUSSION TITLE
Welcome to tinyburg Discussions!

### DISCUSSION THREAD

**Discussion by leonitousconforti** [leonitousconforti/tinyburg#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Ask questions you’re wondering about.
  * Engage with other community members.

**Comment by purepanic** [comment]
How is the secret salt obtained?

**Reply by leonitousconforti** [reply]
Hey @purepanic,

Thanks for taking a look around here! I can't tell you exactly how to retrieve the authentication keys need to connect directly to Nimblebit's game servers. You will need to do a significant amount of reverse engineering to get it. 

If you just want to mess around with the packages here, please take a look at [packages/integrations/core+authproxy](https://github.com/leonitousconforti/tinyburg/tree/main/packages/integrations/core%2Bauthproxy) to see how to use `@tinyburg/core` with tinyburg's open source authentication proxy `@tinyburg/authproxy`. I hope you find it helpful.

A final note, I consider everything here to still be in a alpha state and I have no timeline for a beta release so don't expect anything near a stable release to come anytime soon. Breaking changes and bugs will be frequent. I am working on automatic unit tests for different versions of the game - but they are incomplete. The latest version of the game that I tested was v4.1.0 so understand that modifying your save game data with a later version of the game might introduce errors. If you just want to do raffle/visits/gifts/etc actions and don't care about modifying your save game data then there is no possibility of introducing errors.

Feel free to reach out with any more questions (discord is a better place to reach me, [@leonitous](https://discordapp.com/users/554001005759561758)),
Happy hacking!
